src/javascripts/Views/referencedByView.js

```diff
@@ -0,0 +1,104 @@
+ngapp.controller('referencedByViewController', function($scope, $element, $timeout, htmlHelpers, treeService, referencedByViewService, referencedByViewColumnsService, hotkeyService, nodeSelectionService, treeViewService, treeColumnService, contextMenuService, contextMenuFactory) {
+    // link view to scope
+    $scope.view = $scope.$parent.tab;
+    $scope.view.scope = $scope;
+
+    $scope.allColumns = referencedByViewColumnsService.columns;
+    $scope.contextMenuItems = contextMenuFactory.referencedByViewItems;
+
+    // inherited functions
+    treeService.buildFunctions($scope, $element);
+    treeViewService.buildFunctions($scope, $element);
+    referencedByViewService.buildFunctions($scope);
+    treeColumnService.buildFunctions($scope, '.referenced-by-view', true);
+    nodeSelectionService.buildFunctions($scope, false);
+    hotkeyService.buildOnKeyDown($scope, 'onTreeKeyDown', 'referencedByView');
+
+    $scope.open = function(node) {
+        const record = node.handle ? node.handle : node;
+        $scope.record = record;
+        $scope.syncWithLinkedViews(record);
+    };
+
+    // scope functions
+    $scope.showContextMenu = function(e) {
+        contextMenuService.showContextMenu($scope, e);
+    };
+
+    $scope.toggleAddressBar = function(visible) {
+        $scope.showAddressBar = visible;
```

Comment:
do we really need to use an address bar on the Referenced By view?  I feel like we don't.  Especially if we link it to the record view.